src/Sarif.Driver.UnitTests/TestMessageLogger.cs

```diff
@@ -43,7 +42,7 @@ public void LogMessage(bool verbose, string message)
 
         public void Log(IRuleDescriptor rule, Result result)
         {
-            NoteTestResult(result.Kind, result.Locations[0].AnalysisTarget[0].Uri.LocalPath);
+            NoteTestResult(result.Kind, result.Locations[0].AnalysisTarget.Uri.LocalPath);
         }
```

Comment:
So there are many places where you no longer need to subscript `AnalysisTarget` and `ResultFile`.